content/blog/2023/unintuitive-objects.md

```diff
@@ -0,0 +1,476 @@
+To understand unintuitive objects,
+let's first define
+what *objects* are
+in a digital system.
+
+## Objects on the Web
+An *object* in a web project
+is something that has
+structure, instances, and purpose.
+
+Just as the structure of an object
+in the physical world
+such as a chair
+includes the seat, back, and legs,
+the structure of a digital object
+includes all the stuff
+that makes up that object.
+For example, on a blog,
+an article is an object.
+The structure of an article
+includes the person who wrote the article,
+the title, the images and text,
+the published date and time,
+the updated date and time,
+the category or topic, and more.
+
+Instances are all the specific occurences
+of an object.
+To go back to a physical object,
+instances of the chair object in my house
+would include the pink roller chair I sit in at my desk,
+the small chair my daughter uses in her room,
+the tiny chair in my daughter's dollhouse,
+each chair at the kitchen table.
+
+{{ embed.img(
+  src='blog/2023/chair-object-shop.jpg',
+  alt='',
+  caption='Five instances of a CHAIR object in a bookshop'
+) }}
+
+Instances of a digital article object
+on a blog could include articles like:
+"When to Choose a Progressive Web App,"
+"SQLAlchemy for Django Developers," or
+"FastAPI Path Operations for Django Developers."
+
+{{ embed.img(
+  src='blog/2023/article-object-web.jpg',
+  alt='',
+  caption='Six instances of an ARTICLE object on OddBlog'
+) }}
+
+The purpose of the chair object
+in our example from the physical world
+is to provide people (and dolls)
+a place to sit down.
+The purpose the digital article object
+on the blog is
+to convey information to readers
+about specific topics of interest.
+
+> "All humans think in objects—designers, end-users, and those who
+> painted the caves of Chauvet. So, as we make our user experiences
+> more object-oriented, we also make them more *intuitive*."
+>
+>--*Sophia Prater*, [ooux.com](https://www.ooux.com/what-is-ooux)
+
+With that definition of objects in mind,
+let's look at what makes a digital object *unintuitive*
+for people using the web.
+Though it may seem backward,
+looking at examples of digital objects gone wrong
+is a really impactful way
+to learn how to make them more intuitive.
+
+## Unintuitve Objects on the Web
+There are four type of unintuitive objects:
+Broken, Isolated, Masked, and Shapeshifter.
+To find examples of these unintuitive objects,
+I decided to go spelunkling through the website
+of one of my favorite places to visit --
+the Denver Zoo.
+The photography and graphic design are lovely,
+and what a great excuse
+to pour over pictures of cute animals!
+ANIMAL, HABITAT, and DEMO
```

Comment:
This is a new idea -- maybe add a screenshot showing animals, habitat, and demo? Or move this into the next section, or as a new paragraph with another sentence added? It feels "dangly" in this section.